src/clojure/nrepl/server.clj

```diff
@@ -19,11 +19,19 @@
     (catch Throwable t
       (log t "Unhandled REPL handler exception processing message" msg))))
 
+(defn- liberally-accept
```

Comment:
I think this name is a bit misleading, as the function doesn't really accept anything - it just normalizes the message map. Perhaps something like `normalize-msg` or whatever would be a better name?

Reply:
Let's go with `normalize-msg`. I was alluding to Postel's law with "liberally accept", but normalize is probably more accurate.